chart/templates/deployment.yaml

```diff
@@ -82,90 +82,39 @@ spec:
               value: '{{ ((.Values.prometheus).createTargets) | default "true" }}'
             - name: CREATE_ALERTS
               value: '{{ ((.Values.prometheus).createAlerts) | default "true" }}'
-            - name: K8S_NAMESPACE
```

Comment:
do we not need it in go-sdk? I think it's available, and we should make use of it